studiengaenge/bachelor_informatik.tex

```diff
@@ -65,29 +65,29 @@ \subsection{Prüfungen: wie und wieso?}
 
 Häufig habt ihr pro Prüfung zwei Versuche, es ist also nicht so schlimm, wenn beim ersten Versuch nicht bestanden habt. Einige Dozierende weichen aber von der Standardregelung ab, teilen euch dies allerdings direkt am Anfang des Semesters mit. Informiert euch also am \emph{Anfang des Semesters} immer genau über die Bedingungen! Solltet ihr eine Prüfung auch beim Zweitversuch nicht bestehen, so besteht in bis zu vier Fällen die Möglichkeit, die Prüfung \emph{auf Antrag beim Prüfungsausschuss} ein weiteres Mal zu wiederholen (gilt nicht für Orientierungsprüfung und Bachelorarbeit). Besteht ihr auch diesen Versuch nicht, verliert ihr euren Prüfungsanspruch endgültig und werdet exmatrikuliert. Dies und weitere studienrelevante Vorschriften sind in der Prüfungsordnung geregelt, deshlab lohnt es sich definitiv, die Prüfungsordnung einmal komplett zu lesen. Außerdem kann sich diese leider immer mal wieder ändern, darauf sollte man achten.
 
-\emph{Klausuranmeldungen sind immer verbindlich}. Überlegt euch also gut, ob ihr euch anmeldet. Solltet ihr euch sicher sein, die Klausur nicht zu bestehen, empfiehlt es sich, sich nicht anzumelden und die Vorlesung im nächsten Jahr noch einmal zu hören. Das Schöne an eurem Studiengang ist, dass die Noten der Grundpflichtmodule (\gls{IPI}, \gls{IPK}, \gls{ITI}, \gls{LA} 1 und \gls{Ana} 1 bzw. \gls{MafIn} 1 und 2) nicht in eure Abschlussnote zählen. Das heißt, dass, auch wenn die ersten Semester mit der ganzen Mathematik vielleicht etwas hart sind, eure Abschlussnote darunter nicht unbedingt zu leiden hat.
+\emph{Klausuranmeldungen sind immer verbindlich}. Überlegt euch also gut, ob ihr euch anmeldet. Solltet ihr euch sicher sein, die Klausur nicht zu bestehen, empfiehlt es sich, sich nicht anzumelden und die Vorlesung im nächsten Jahr noch einmal zu hören. Das Schöne an eurem Studiengang ist, dass die Noten der Grundpflichtmodule (\gls{IPI}, \gls{IPK}, \gls{ITE}, \gls{LA} 1 und \gls{Ana} 1 bzw. \gls{MafIn} 1 und 2) nicht in eure Abschlussnote zählen. Das heißt, dass, auch wenn die ersten Semester mit der ganzen Mathematik vielleicht etwas hart sind, eure Abschlussnote darunter nicht unbedingt zu leiden hat.
```

Comment:
Ab der neuen PO zählen alle Module in die Abschlussnote, aber man kann zwei streichen, oder?